app.js

```diff
@@ -1,13 +1,30 @@
 const express = require("express");
-const { getTopics } = require("./controllers/controller");
+const { getTopics, getArticleByID } = require("./controllers/controller");
 
 const app = express();
 app.use(express.json());
```

Comment:
You don't need this line yet as you're not doing a post request :), so I would remove for now